tests/integration/diff_test.go

```diff
@@ -1725,7 +2629,7 @@ func Test_Diff_Unmasked_NewerThan3x(t *testing.T) {
 }
 
 func Test_Diff_PluginUpdate_NewerThan38x(t *testing.T) {
```

Comment:
We could change this test's name to 

```suggestion
func Test_Diff_PluginUpdate_38x(t *testing.T) {
```

given the updated constrain.